invenio_access/upgrades/access_2015_05_06_accROLE_accACTION_accARGUMENT_id.py

```diff
@@ -70,7 +70,9 @@ def do_upgrade():
         CHANGE COLUMN "id_accARGUMENT" "id_accARGUMENT" INT(15) NULL ,
         CHANGE COLUMN "argumentlistid" "argumentlistid" MEDIUMINT(8) NULL ,
         ADD COLUMN "id" INT(15) UNSIGNED NOT NULL AUTO_INCREMENT,
-        DROP PRIMARY KEY,
+        DROP KEY id_accROLE,
```

Comment:
I'm not sure if these indexes should be removed. Can you paste your current `describe table accROLE_accACTION_accARGUMENT`?